src/components/PlayerSearch.vue

```diff
@@ -105,6 +92,13 @@ export default {
     },
   },
   methods: {
+    getArray(num) {
```

Comment:
Please add a comment explaining what this method does and where the values come from.